README.md

```diff
@@ -7,25 +7,23 @@ WASM runtime.
 
 ### Dependencies
 
-- Go 1.22.6
-- Tinygo v0.32.0
+- Go 1.23.8
+- Tinygo v0.34.0
```

Comment:
this is not strictly necessary for this PR, but I realized that I think a few folks are already using 0.34. This might be a bit of a moot point since I think this was addressed in #10